services/dbt/target/run/data_pipelines/models/datenspende/daily_vital_features/daily_aggregates_of_vitals.sql

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
is it good practice to commit the /target directory of dbt? from my understanding this gets filled when `dbt run` is executed with the single sql statements to actually run, and is therefore re-generated each run?

Reply:
The sql is also compiled when building the docs. But since it is not needed to render them, we shouldn't commit it to keep the diffs small.